audio/out/ao_pulse.c

```diff
@@ -465,6 +465,9 @@ static int init(struct ao *ao)
         goto unlock_and_fail;
     }
 
+    ao->device_buffer = pa_stream_get_buffer_attr(priv->stream)->tlength /
```

Comment:
Unfortunately https://freedesktop.org/software/pulseaudio/doxygen/stream_8h.html#a9a3c3e78eafb28cce3a16cef2b68a385 doesn't say that it will never return nullptr, so this would have to be handled.